src/plugins/vis_builder/public/visualizations/table/components/table_viz_options.tsx

```diff
@@ -3,14 +3,12 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { get } from 'lodash';
-import React, { useCallback, useEffect, useMemo } from 'react';
+import React, { useCallback } from 'react';
 import { i18n } from '@osd/i18n';
 import { FormattedMessage } from '@osd/i18n/react';
 import produce from 'immer';
 import { Draft } from 'immer';
 import { EuiIconTip } from '@elastic/eui';
-import { search } from '../../../../../data/public';
```

Comment:
Did we just miss this import cleanup previously?

Reply:
Yes, I didn't find out this cleanup issue before.